extensions/json-schema/rest/src/main/java/org/apache/unomi/schema/rest/JsonSchemaEndPoint.java

```diff
@@ -65,6 +68,18 @@ public Set<String> getInstalledJsonSchemaIds() {
         return schemaService.getInstalledJsonSchemaIds();
     }
 
+    /**
+     * Get a schema by it's id
+     *
+     * @param id of the schema
+     * @return Json schema matching the id
+     */
+    @POST
+    @Path("/queryId")
+    public String getSchema(String id) {
+        return schemaService.getSchema(id).getSchema().replace("\n", "");
```

Comment:
be careful of NPE here what happen if: schemaService.getSchema(id) is returning null ? :)